internal/pkg/mapper/template/jsonnetfiles/jsonnetfiles_test.go

```diff
@@ -11,7 +12,7 @@ import (
 
 func createStateWithMapper(t *testing.T, jsonnetCtx *jsonnet.Context) *state.State {
 	t.Helper()
-	d := dependencies.NewMocked(t)
+	d := dependencies.NewMocked(t, context.Background())
 	mockedState := d.MockedState()
```

Comment:
`ctx` is passed to the dependencies construtors, so there will be always only one root context in tests.